### DISCUSSION TITLE
funds not showing up on solana

### DISCUSSION THREAD

**Discussion by odinfree** [wormhole-foundation/wormhole#510] [Problems / bugs] [answered]
Hi I transferred funds from ethereum to solana phantom wallet but its not coming trough and recovering does not work. 

0x4fd2631054e8255ad5a4b52ba320406dac4cba67b2095caff99443ac2f6e89bc
is the transaction ID. 

please help.

**Comment by leoluk** [comment, marked as answer]
You should be able to use the Redeem feature on https://www.wormholebridge.com/ to complete the transfer.
